release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -419,6 +419,12 @@ For more information, see xref:../nodes/clusters/nodes-cluster-enabling-features
 [id="ocp-release-notes-installation-and-update_{context}"]
 === Installation and update
 
+[id="ocp-4.18-installation-and-update-byoip-ovn_{context}"]
+==== Bring your own IPv4 (BYOIP) join subnet feature is enabled on `ovn-kubernetes` for Virtual Private Cloud (VPC)
```

Comment:
Searching the existing document, `BYOIP` refers to _Bring your own public IPv4 addresses_ [1], which is for publicIpv4Pool field, but internalJoinSubnet uses internal/private addresses, I think we can update the title like _Add support to configure IPv4 subnet to customize internal OVN network_ to reduce the confusion. 

@patrickdillon do you have any idea about the title?

[1] https://docs.openshift.com/container-platform/4.17/installing/installing_aws/installation-config-parameters-aws.html

Reply:
Maybe something like

Install-time configuration for the `ovn-kubernetes` join subnet is now enabled